public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -173,7 +202,28 @@ export const DataConnection = (props: any) => {
       id: 'associated_objects',
       name: 'Associated Objects',
       disabled: false,
-      content: <AssociatedObjectsTab associatedObjects={mockAssociatedObjects} />,
+      content: (
+        <AssociatedObjectsTab
+          associatedObjects={mockAssociatedObjects}
+          datasource={datasourceDetails}
+          cachedDatabases={cachedDatabases}
+          databasesLoadStatus={databasesLoadStatus}
+          selectedDatabase={selectedDatabase}
+          setSelectedDatabase={setSelectedDatabase}
+          cachedTables={cachedTables}
+          setCachedTables={setCachedTables}
+          tablesLoadStatus={tablesLoadStatus}
+          startLoadingTables={startLoadingTables}
+          tablesIsLoading={tablesIsLoading}
+          cachedAccelerations={cachedAccelerations}
+          setCachedAccelerations={setCachedAccelerations}
+          accelerationsLoadStatus={accelerationsLoadStatus}
+          startLoadingAccelerations={startLoadingAccelerations}
+          accelerationsIsLoading={accelerationsIsLoading}
+          isFirstTimeLoading={isFirstTimeLoading}
+          isRefreshing={isRefreshing}
```

Comment:
There are just too many props passed here. Does it makes sense to move them into the `AssociatedObjectsTab` and keep only necessary ones here?